Gems/Atom/Feature/Common/Editor/Scripts/ColorGrading/__init__.py

```diff
@@ -0,0 +1,15 @@
+# coding:utf-8
+#!/usr/bin/python
+#
+# Copyright (c) Contributors to the Open 3D Engine Project
```

Comment:
this and a few others appear to be missing the line about looking for LICENSE at the root of the project

Reply:
I will go figure out what is missed